packages/common/src/utils/SafeLocalStorage.ts

```diff
@@ -29,7 +29,7 @@ export const SafeLocalStorageKeys = {
   SOLANA_CAIP_CHAIN: '@appkit/solana_caip_chain',
   ACTIVE_CAIP_NETWORK_ID: '@appkit/active_caip_network_id',
   CONNECTED_SOCIAL: '@appkit/connected_social',
-  CONNECTED_SOCIAL_USERNAME: '@appkit/connected_social_username',
+  CONNECTED_SOCIAL_USERNAME: '@appkit-wallet/SOCIAL_USERNAME',
```

Comment:
Do we need to change the local storage name ?

Reply:
`@appkit-wallet/SOCIAL_USERNAME` is set in wallet package. I think it is good to keep that naming. This will also fix the issue for users that already have this value in their localstorage